sqlmesh/core/snapshot/definition.py

```diff
@@ -327,10 +327,8 @@ class Snapshot(PydanticModel, SnapshotInfoMixin):
 
     Args:
         name: The snapshot name which is the same as the model name and should be unique per model.
-
         fingerprint: A unique hash of the model definition so that models can be reused across environments.
-        physical_schema: The physical schema that the snapshot is stored in.
-        model: Model object that the snapshot encapsulates.
+        node: Model object that the snapshot encapsulates.
```

Comment:
I guess it's `Node object` now

Reply:
Yeah I think there a lot of these mismatches in docstrings now. I'm just cleaning up the ones I notice.